README.md

```diff
@@ -1,2 +1,71 @@
+| [notaryproject.dev](https://github.com/notaryproject/notaryproject.dev) | This repository contains the source code and content for the [Notary Project website](https://notaryproject.dev). | 
+| [notation](https://github.com/notaryproject/notation) | This repository contains the source code for the convenient CLI implementation of the new Notary Project signing and verification flow. |
+| [notation-go](https://github.com/notaryproject/notation-go) | This repository contains the source code for the convenient Golang library implementation of the new Notary Project signing and verification flow. |
+| [notation-core-go](https://github.com/notaryproject/notation-core-go) | This repository contains the source code for the Golang library implementation of the Notary Project signature (hereafter "Notary Project signature")  specification and wrapping (COSE and JWS). |
+| [roadmap](https://github.com/notaryproject/roadmap) | This repository is intended for keeping track of development activities in the Notary Project. It may be retired in the future as feature request and milestones are moved to the appropriate repositories. |
+| [tuf](https://github.com/notaryproject/tuf) | This repository is intended for prototyping the storage of TUF metadata in OCI-compliant registries. It is not under active development at the moment but there are plans to revive it in the future. |
+
+## Project Status
+
+The Notary Project is in active development. The latest release announcements are published on the [Notary Project blog](https://notaryproject.dev/blog/). The Notary Project community uses the [project board](https://github.com/orgs/notaryproject/projects/10) for project planning and status tracking. You can also use GitHub milestones to track the progress of each repository:
+
+- [The Notary Project specification milestones](https://github.com/notaryproject/notaryproject/milestones)
+- [notation CLI milestones](https://github.com/notaryproject/notation/milestones)
+- [notation-go library milestones](https://github.com/notaryproject/notation-go/milestones)
+- [notation-core-go library milestones](https://github.com/notaryproject/notation-core-go/milestones)
+- [notary milestones](https://github.com/notaryproject/notary/milestones)
+
+You can also check the release pages of each repository for the latest release binaries:
+
+- [The Notary Project specification releases](https://github.com/notaryproject/notaryproject/releases)
+- [notation CLI releases](https://github.com/notaryproject/notation/releases)
+- [notation-go library releases](https://github.com/notaryproject/notation-go/releases)
+- [notation-core-go library releases](https://github.com/notaryproject/notation-core-go/releases)
+- [notary releases](https://github.com/notaryproject/notary/releases)
+
+## Security
+
+The Notary Project has had several public security audits:
```

Comment:
suggest we flip in reverse chronological order with the latest security audit shown first

Reply:
Updated the section and reversed the order.